[main] EVA unit test fails randomly on Mac

https://jenkins-ci.tribler.org/job/GH_Tribler_PR_Tests/job/PR_mac_pytest/1840/testReport/junit/src.tribler-core.tribler_core.modules.remote_query_community.tests.test_eva_protocol/TestEVA/test_one_megabyte_transfer/

```python
Error Message

TypeError: 'NoneType' object is not subscriptable

Stacktrace

self = <test_eva_protocol.TestEVA testMethod=test_one_megabyte_transfer>

    @pytest.mark.timeout(15)
    async def test_one_megabyte_transfer(self):
        data_size = 1024 * 1024
        data = os.urandom(1), os.urandom(data_size), random.randrange(0, 256)
    
        self.overlay(0).eva_send_binary(self.peer(1), *data)
    
        await drain_loop(asyncio.get_event_loop())
    
>       assert len(self.overlay(1).most_recent_received_data[1]) == data_size
E       TypeError: 'NoneType' object is not subscriptable

src/tribler-core/tribler_core/modules/remote_query_community/tests/test_eva_protocol.py:137: TypeError
```

This could be related to: https://jenkins-ci.tribler.org/job/GH_Tribler_PR_Tests/job/PR_mac_pytest/1845/console

```python
_______________ TestEVA.test_wrong_message_order_and_wrong_nonce _______________
[gw6] darwin -- Python 3.8.4 /usr/local/opt/python@3.8/bin/python3.8
/usr/local/lib/python3.8/site-packages/asynctest/case.py:297: in run
    self._run_test_method(testMethod)
/usr/local/lib/python3.8/site-packages/asynctest/case.py:354: in _run_test_method
    self.loop.run_until_complete(result)
/usr/local/lib/python3.8/site-packages/asynctest/case.py:224: in wrapper
    return method(*args, **kwargs)
/usr/local/Cellar/python@3.8/3.8.4/Frameworks/Python.framework/Versions/3.8/lib/python3.8/asyncio/base_events.py:603: in run_until_complete
    self.run_forever()
/usr/local/lib/python3.8/site-packages/asynctest/case.py:224: in wrapper
    return method(*args, **kwargs)
/usr/local/Cellar/python@3.8/3.8.4/Frameworks/Python.framework/Versions/3.8/lib/python3.8/asyncio/base_events.py:570: in run_forever
    self._run_once()
/usr/local/Cellar/python@3.8/3.8.4/Frameworks/Python.framework/Versions/3.8/lib/python3.8/asyncio/base_events.py:1823: in _run_once
    event_list = self._selector.select(timeout)
/usr/local/lib/python3.8/site-packages/asynctest/selector.py:299: in select
    return self._selector.select(timeout)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <selectors.KqueueSelector object at 0x13542f310>
timeout = 0.009756768999992005

    def select(self, timeout=None):
        timeout = None if timeout is None else max(timeout, 0)
        max_ev = len(self._fd_to_key)
        ready = []
        try:
>           kev_list = self._selector.control(None, max_ev, timeout)
E           Failed: Timeout >10.0s

/usr/local/Cellar/python@3.8/3.8.4/Frameworks/Python.framework/Versions/3.8/lib/python3.8/selectors.py:558: Failed
```

--- a/src/tribler-core/tribler_core/modules/remote_query_community/tests/test_eva_protocol.py
+++ b/src/tribler-core/tribler_core/modules/remote_query_community/tests/test_eva_protocol.py
@@ -22,7 +22,7 @@ from tribler_core.modules.remote_query_community.eva_protocol import (
 )
 
 # fmt: off
-
+PYTEST_TIMEOUT_IN_SEC = 60
 
 TEST_DEFAULT_TERMINATE_INTERVAL_IN_SEC = 1
 TEST_DEFAULT_RETRANSMIT_INTERVAL_IN_SEC = 0.3
@@ -125,8 +125,11 @@ class TestEVA(TestBase):
         assert len(self.overlay(0).sent_data[self.peer(1)]) == 0
         assert len(self.overlay(1).received_data[self.peer(0)]) == 0
 
-    @pytest.mark.timeout(15)
+    @pytest.mark.timeout(PYTEST_TIMEOUT_IN_SEC)
     async def test_one_megabyte_transfer(self):
+        self.overlay(0).eva_protocol.terminate_by_timeout_enabled = False
+        self.overlay(1).eva_protocol.terminate_by_timeout_enabled = False
+
         data_size = 1024 * 1024
         data = os.urandom(1), os.urandom(data_size), random.randrange(0, 256)
 
@@ -137,7 +140,7 @@ class TestEVA(TestBase):
         assert len(self.overlay(1).most_recent_received_data[1]) == data_size
         assert self.overlay(1).most_recent_received_data == data
 
-    @pytest.mark.timeout(15)
+    @pytest.mark.timeout(PYTEST_TIMEOUT_IN_SEC)
     async def test_termination_by_timeout(self):
         # breaks "on_data" function in community2 to make this community silent
 
@@ -232,6 +235,7 @@ class TestEVA(TestBase):
         assert not self.overlay(2).eva_protocol.outgoing
         assert len(self.overlay(0).sent_data[self.peer(2)]) == 0
 
+    @pytest.mark.timeout(PYTEST_TIMEOUT_IN_SEC)
     async def test_duplex(self):
         count = 100
         block_size = 10
@@ -262,7 +266,11 @@ class TestEVA(TestBase):
         assert len(self.overlay(1).sent_data[self.peer(0)]) == 1
         assert len(self.overlay(1).received_data[self.peer(0)]) == 1
 
+    @pytest.mark.timeout(PYTEST_TIMEOUT_IN_SEC)
     async def test_multiply_send(self):
+        self.overlay(0).eva_protocol.terminate_by_timeout_enabled = False
+        self.overlay(1).eva_protocol.terminate_by_timeout_enabled = False
+
         data_set_count = 100
         data_size = 1024
 
@@ -275,6 +283,7 @@ class TestEVA(TestBase):
         assert self.overlay(1).received_data[self.peer(0)] == data_list
         assert not self.overlay(0).eva_protocol.scheduled
 
+    @pytest.mark.timeout(PYTEST_TIMEOUT_IN_SEC)
     async def test_multiply_duplex(self):
         data_set_count = 10
 
@@ -315,7 +324,7 @@ class TestEVA(TestBase):
 
         assert data_sets_checked == 6
 
-    @pytest.mark.timeout(20)
+    @pytest.mark.timeout(PYTEST_TIMEOUT_IN_SEC)
     async def test_survive_when_multiply_packets_lost(self):
         self.overlay(0).eva_protocol.terminate_by_timeout_enabled = False
         self.overlay(1).eva_protocol.terminate_by_timeout_enabled = False
@@ -371,7 +380,7 @@ class TestEVA(TestBase):
         assert len(self.overlay(1).received_data[self.peer(0)]) == data_set_count
         assert self.overlay(1).received_data[self.peer(0)] == data
 
-    @pytest.mark.timeout(15)
+    @pytest.mark.timeout(PYTEST_TIMEOUT_IN_SEC)
     async def test_dynamically_changed_window_size(self):
         window_size = 5
 
@@ -427,7 +436,7 @@ class TestEVA(TestBase):
         assert isinstance(self.overlay(0).most_recent_received_exception, TransferException)
         assert isinstance(self.overlay(1).most_recent_received_exception, SizeLimitException)
 
-    @pytest.mark.timeout(10)
+    @pytest.mark.timeout(PYTEST_TIMEOUT_IN_SEC)
     async def test_wrong_message_order_and_wrong_nonce(self):
         self.overlay(0).eva_protocol.terminate_by_timeout_enabled = False
         self.overlay(1).eva_protocol.terminate_by_timeout_enabled = False

